release/_git.sh

```diff
@@ -202,16 +204,9 @@ function create_branch {
 		original_branch_name="${LIFERAY_RELEASE_GIT_PREV_REF}"
 	fi
 
-	local last_commit=$(\
-		curl \
-			"https://api.github.com/repos/liferay/liferay-portal-ee/git/refs/heads/${original_branch_name}" \
-			--fail \
-			--header "Authorization: token ${LIFERAY_RELEASE_GITHUB_TOKEN}" \
-			--max-time 10 \
-			--retry 3 \
-			--silent)
+	local last_commit=$(invoke_get_github_api "https://api.github.com/repos/liferay/liferay-portal-ee/git/refs/heads/${original_branch_name}")
```

Comment:
@kiwm, can we rename to `invoke_github_get_api` and `invoke_github_post_api`?

Reading this looks clearer. Sorry. :grin: